src/utils/fetchUser.utils.ts

```diff
@@ -0,0 +1,35 @@
+import { User, Token } from 'devu-shared-modules'
+
+import { UserState } from 'redux/types/user.types'
+
+import RequestService from 'services/request.service'
+
+import { decodeAccessToken } from 'utils/authentication.utils'
+
+/**
+ * Function to grab all the data that the application needs to initialize itself
+ *
+ * Right now that means grabbing yourself an access token as well as the user info for yourself
+ */
+async function fetchUserInfo(): Promise<UserState> {
+  const loginResponse: Token = await RequestService.get('/api/login', { credentials: 'include' })
```

Comment:
We should change the endpoint from `/api/login` to `/api/token` or similar for sake of clarity

Reply:
that would require changing the entire login controller to token cause /login is the root name on the route in the api. But we certainly can do that.

Reply:
I guess we can cross that bridge when we get there - im not super concerned with it at the moment and the future refactor shouldnt be too difficult if we want token operations to be on a separate route.